torch/_inductor/config.py

```diff
@@ -187,7 +187,7 @@ class trace:
     info_log = False
 
     # Save input FX graph (post decomps)
-    fx_graph = True
+    fx_graph = False
```

Comment:
🤔

Reply:
This was just since the FX graph dumping code is broken with dynamic shapes.